TundraTD/Assets/Scripts/City/Building/Upgrades/FireTowerUpgrades/MeteorLandingStanUpgrade.cs

```diff
@@ -0,0 +1,20 @@
+using City.Building.ElementPools;
+using Spells;
+using UnityEngine;
+
+namespace City.Building.Upgrades.FireTowerUpgrades
+{
+    public class MeteorLandingStanUpgrade : IUpgrade
```

Comment:
A typo, not Stan, "Stun" actually
```suggestion
    public class MeteorLandingStunUpgrade : IUpgrade
```